ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -170,6 +169,8 @@ conformance_test(
     ],
 )
 
+# TODO SoX: Create conformance test with --static-time.
```

Comment:
This PR adds support for static time mode, I think it should also add a corresponding conformance test.

Reply:
I tried to. But the test wouldn't fail no matter if I turn static-time on or off. Will investigate it async and leaving a TODO in here.

Reply:
Ok, I've created a static-time test as it is created in sandbox-classic. At the moment, it has little value, since the ledger-api-test-tool doesn't have negative tests for the time mode (when using `CommandDeduplicationIT`, all tests pass regardless of the time mode set on the server/test-tool). Since creating negative test cases for static-time is not in the scope this task, I'll merge like this and create a separate task for investigation.